build.sbt

```diff
@@ -0,0 +1,40 @@
+val scalaOptions = Seq(
```

Comment:
These should already be configured via `sbt-auto-build` which in turn uses `sbt-settings`.
The only one I don't see is `-language` but I'm not sure if a wildcard actually works there (it would seem a little dangerous if it does).  The docs [scala compiler options](https://docs.scala-lang.org/overviews/compiler-options/index.html) do not mention wildcard support - but this could always be an oversight.

Reply:
Removed all existing scalac options. `-language` is the only setting that is not included in `sbt-auto-build`. But it appears too dangerous to include as a wildcard. So I have removed that setting and introduced `-feature` which will surface detailed warnings when an advanced featured is used without relevant settings/imports.